packages/js/src/generators/library/library.ts

```diff
@@ -599,7 +617,7 @@ async function normalizeOptions(
     }
   }
 
-  // This is to preserve old behaviour, buildable: false
+  // This is to preserve old behavior, buildable: false
```

Comment:
How dare you 🇬🇧

(JK using the American spelling consistently is what I usually intend to go for anyway)

Reply:
Hah, I didn't realize this was a legitimate British spelling! I just took the autocorrect suggestion from vscode and didn't think about it 😅